bigquery_etl/cli/glean_usage.py

```diff
@@ -14,6 +14,9 @@
     baseline_clients_first_seen,
     baseline_clients_last_seen,
     events_unnested,
+    clients_daily_metrics,
+    clients_last_seen_metrics,
+    clients_last_seen_joined,
```

Comment:
The proposal calls these queries `clients_daily_metrics` and `clients_last_seen_metrics`. I was wondering why they are not called `metrics_clients_daily` and `metrics_clients_last_seen`?